github/ci/prow-deploy/kustom/base/configs/current/labels/labels.yaml

```diff
@@ -42,12 +42,6 @@ default:
       description: Denotes a PR that will be considered when it comes time to generate release notes.
       prowPlugin: release-note
       addedBy: prow
-    - color: c2e0c6
```

Comment:
Please don't remove this one, this is used by the release-note prow plugin.

From the docs:

> PRs that contain 'action required' in their 'release-note' block are given the label 'release-note-action-required' to indicate that the PR introduces potentially breaking changes that necessitate user action before upgrading to the release.

[source](https://prow.ci.kubevirt.io/plugins)

Reply:
Thanks Daniel, I'll update the doc PR